### PR TITLE
Examples: add a Photosensitive Epilepsy warning

### LINKED ISSUE #20968: Potential Photosensitive epilepsy at demo pages
**Describe the bug**

This [demo page](https://threejs.org/examples/webgl_postprocessing_glitch) contains extremely strong blinks and flashes, which may cause a potential Photosensitive epilepsy. Better to add a warning at the entry of this demo.

**To Reproduce**

1. Go to 'https://threejs.org/examples/webgl_postprocessing_glitch'

***Code***

none

***Live example***

none

**Expected behavior**

Add a warning at the entry of this demo page.

**Screenshots**

none

**Platform:**

none

### PR DESCRIPTION
**Pull request by FrodoLuo**
Related issue: Fixed #20968

**Description**

Added a overlay to warn the potential Photosensitive Epilepsy

### CODE REVIEW

File: examples/webgl_postprocessing_glitch.html [hunk 1]
```diff
@@ -6,8 +6,22 @@
...
+		}
+	</style>
 	<body>
 
+		<div id="overlay">
+			<h3>Photosensitive Epilepsy Warning</h3>
+			<p class="warning">
+				This demo contains strong and frequent flashes which may trigger a potential <strong>Photosensitive Epilepsy.</strong>
+				<br/>
+				If you got any uncomfortable, press <strong>Ctrl + W</strong> or <strong>Command + W</strong> to close this page.
```
Comment (Mugen87) [inline examples/webgl_postprocessing_glitch.html:21]: I don't think we have to explain user how to close a browser tab 😉 .

**Comment by Mugen87** [comment]
For testing: https://raw.githack.com/FrodoLuo/three.js/photosensitive_epilesy_warning/examples/webgl_postprocessing_glitch.html

File: examples/webgl_postprocessing_glitch.html (hunk 1 above)
Comment (FrodoLuo) [reply examples/webgl_postprocessing_glitch.html:21]: I see... Let me commit another change

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**